### DISCUSSION TITLE
Develop a button that pulls the latest RuneScape update, instead of having to close and reopen RuneLite

### DISCUSSION THREAD

**Discussion by peroa** [runelite/runelite#15551] [Q&A] [answered]
I keep RuneLite open most of the time and I don't regularly shutdown/restart my computer, only putting it to sleep at night. Yeah, this is poor hardware hygiene, so be it. Because RuneLite stays running indefinitely, I often run into a RuneScape update that blocks me from logging in: "RuneScape has been updated! Please reload this page."
![image](https://user-images.githubusercontent.com/28541551/186463616-dc25655c-5871-4e86-92f5-8f6885b2323c.png)

The only way I'm aware of to pull in a new RuneScape update is to close RuneLite and reopen it. I think it would be a nice enhancement to have a button in the Info tab (perhaps right under the version numbers) that pulls in the latest RuneScape update(s) without restarting RuneLite. My intention for this enhancement is to complete this update task in fewer clicks and ensure that the RuneLite window stays in the same screen location. I've had instances where, after restarting RuneLite, I have to move and resize the window to get it back to a similar screen location where it was before.

This is definitely a nitpick and a request for a trivial enhancement, but it got me thinking how convenient the refresh button is on an internet browser.

**Comment by Enriath** [comment, marked as answer]
I don't think our stance on this has changed since #6405 was opened nearly 3 years ago.

**Reply by peroa** [reply]
Fair enough, I did search for previous discussions but didn't try the term "Refresh". Thanks for the reply!

**Comment by geheur** [comment]
Have you tried "Remember client position" in "Runelite" plugin?
